src/components/Icon.tsx

```diff
@@ -294,20 +295,18 @@ export const Icon = ({ name, className, size }: IconProps) => {
 				<svg
 					className={className}
 					viewBox="0 0 20 20"
-					fill="none"
+					fill="currentColor"
```

Comment:
but then seeing changes to an older icon makes me think we should have them in there for chromatic

Reply:
no, let's focus on finishing everything this week. keep the meeting postponed but do have it